### PR TITLE
Final implementation of the landing page

### PR DESCRIPTION
**Pull request by KattaManisha**

### CODE REVIEW

**Review by KattaManisha — COMMENTED** [review]
The footer seems to be empty, just  add about the Estimate IT and it's description and let's try to maintain same footer in every page

**Review by KattaManisha — COMMENTED** [review]
You are good to merge the code in the main!

File: templates/clothing.html [hunk 1]
```diff
@@ -86,9 +86,9 @@ <h5 class="card-title">Adidas Red Hoodie</h5>
             <a href="/fetch-home"
                 ><button type="button" class="btn btn-warning" id="compareButton">
               Compare Prices
-            </button><a href="/fetch-home"
```
Comment (TaariqMansurie) [inline templates/clothing.html:89]: Good that you removed the unnecessary href tag as it was causing an issue !

**Review by TaariqMansurie — APPROVED** [review]
Good job !

**Pull request merged**